src/services/item/plugins/publication/published/repositories/itemPublished.ts

```diff
@@ -53,7 +54,7 @@ export class ItemPublishedRepository {
     });
   }
 
-  async getForMember(memberId: Member['id']): Promise<Item[]> {
+  async getForMember(accountId: Account['id']): Promise<Item[]> {
```

Comment:
Rename? Only members can get their own published items.

Reply:
Is this function only used for tests cases ? I didn't been able to find anything else..